From c7f93fb7f7ad8e3b04f9283f6474a919f31fe5a7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Fri, 10 Sep 2010 19:10:36 +0000
Subject: [PATCH 118/160] gettext docs: the Git::I18N Perl interface
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change po/README's "Marking strings for translation" section so that
it covers the Git::I18N Perl interface.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 po/README |   11 +++++++++++
 1 files changed, 11 insertions(+), 0 deletions(-)

diff --git a/po/README b/po/README
index a93c88d..66ab35f 100644
--- a/po/README
+++ b/po/README
@@ -168,3 +168,14 @@ Shell:
    command to be translated) for examples is also useful:
 
        git log --reverse -p --grep=gettextize git-am.sh
+
+Perl:
+
+ - The Git::I18N module provides a limited subset of the
+   Locale::Messages functionality, e.g.:
+
+       use Git::I18N;
+       print __("Welcome to Git!\n");
+       printf __("The following error occured: %s\n"), $error;
+
+   Run `perldoc perl/Git/I18N.pm` for more info.
-- 
1.7.3.1.50.g1e633

